src/map/utils/mobutils.cpp

```diff
@@ -141,7 +141,6 @@ namespace mobutils
         bool isEventMob = PMob->m_Type & MOBTYPE_EVENT && !PMob->isInDynamis();
         bool isBattlefieldMob = PMob->m_Type & MOBTYPE_BATTLEFIELD && !PMob->isInDynamis();
```

Comment:
Can probably remove the edits from this file? The formatting changes don't work for styling